packages/editor/package.json

```diff
@@ -38,6 +38,7 @@
 		"@wordpress/api-fetch": "file:../api-fetch",
 		"@wordpress/blob": "file:../blob",
 		"@wordpress/block-editor": "file:../block-editor",
+		"@wordpress/block-library": "file:../block-library",
```

Comment:
Suggestion (blocking): Let's not add the whole package as a dependency just for a single util method.